config.py

```diff
@@ -570,17 +570,20 @@ def get_system_stats():
 
     platf = sys.platform
 
-    if Platform.is_linux(platf):
-        output, _, _ = get_subprocess_output(['grep', 'model name', '/proc/cpuinfo'], log)
-        systemStats['cpuCores'] = len(output.splitlines())
-
-    if Platform.is_darwin(platf):
-        output, _, _ = get_subprocess_output(['sysctl', 'hw.ncpu'], log)
-        systemStats['cpuCores'] = int(output.split(': ')[1])
-
-    if Platform.is_freebsd(platf):
-        output, _, _ = get_subprocess_output(['sysctl', 'hw.ncpu'], log)
-        systemStats['cpuCores'] = int(output.split(': ')[1])
+    try:
+        if Platform.is_linux(platf):
+            output, _, _ = get_subprocess_output(['grep', 'model name', '/proc/cpuinfo'], log)
+            systemStats['cpuCores'] = len(output.splitlines())
+
+        if Platform.is_darwin(platf):
+            output, _, _ = get_subprocess_output(['sysctl', 'hw.ncpu'], log)
+            systemStats['cpuCores'] = int(output.split(': ')[1])
+
+        if Platform.is_freebsd(platf):
+            output, _, _ = get_subprocess_output(['sysctl', 'hw.ncpu'], log)
+            systemStats['cpuCores'] = int(output.split(': ')[1])
```

Comment:
not your code, but seems like this could be written as:
```python
if Platform.is_linux(platf):
...
elif Platform.is_darwin(platf) or Platform.is_freebsd(platf):
...
```